Insteon documentation update for button event entities (#44960)
home-assistant/home-assistant.github.io: source/_integrations/insteon.markdown

        group: 25
```

## Button event entities

> Dimmers, switches, keypads have a **Button Press** event entity for the main button on the device. Keypads also have **Button X Press** event entities where **X** is the button **A** through **H**.

Reviewer: ```suggestion
Dimmers, switches, keypads have a **Button press** event entity for the main button on the device. Keypads also have **Button X press** event entities where **X** is the button **A** through **H**.
```

Each button press event has 4 event_type values: **on**, **off**, **on_fast** and **off_fast**.

> - **on**: A single ON press

Reviewer: ```suggestion
- **on**: Single press of the On button
```

> - **off**: A single OFF press

Reviewer: ```suggestion
- **off**: Single press of the Off button
```

> - **on_fast**: A quick double ON tap

Reviewer: ```suggestion
- **on_fast**: Quick double-press of the On button
```

> - **on_fast**: A quick double ON tap

Reviewer: What does on/off fast actually do? Turn it on off faster, without dimming? Or is it just a programmable button?

Author: If it's a dimmer, it turns on/off instantly.  For a switch, it's no different.  "fast" is the term Insteon uses.

I use the double press fast on/off as a automation trigger.  For example, single press off turns that light off.  Double off press turns all the lights off in the area.  Same for on press.  These events also happened whether the light/switch is already in the on or off state, so they make handy automation triggers throughout the house.

Reviewer: I see. Would it be worth adding a note there about how it could be used?

Author: I added a real world example.  (messed up the PR merge in the process, but it's there now)

Reviewer: Ah, nice! Thank you.

> - **off_fast** A quick double OFF tap

Reviewer: ```suggestion
- **off_fast**: Quick double-press of the Off button
```

To trigger an automation with a button press event, select the device's button press entity and the event_type attribute.

> Example: To trigger an automation from "double-tapping" a switch on, select the **Button Press** entity, the **Event Type** attribute and the **on_fast** value for the **To** field.

Reviewer: ```suggestion
Example: To trigger an automation from double-pressing an on switch, select the **Button press** entity, the **Event type** attribute and the **on_fast** value for the **To** field.
```
